subsys/partition_manager/Kconfig

```diff
@@ -137,6 +137,16 @@ endif
 config BUILD_WITH_TFM
 	bool
 	select PM_SINGLE_IMAGE
+	select ENTROPY_CC3XX
+	select NORDIC_SECURITY_BACKEND
+	select MBEDTLS_X509_LIBRARY
```

Comment:
Could these be `imply` instead? Currently it is not possible to create any small builds since these are selected (and hence impossible to disable).

Reply:
Agree, imply would be fine. We already use imply in another definition of BUILD_WITH_TFM. I do not thing that these need to be selected by default. E.g. does not make sense when we do not care at all about crypto.

Reply:
For both the nRF5340 and the nRF9160 you are required to use ENTROPY_CC3XX and NORDIC_SECURITY_BACKEND to get entropy. Both of those needs to be selected for a "minimal build" given that RNG is required for core functionality.

MBEDTLS_X509_LIBRARY should only impact NS build code side, and it doesn't require any static memory. The cost is only when you call the APIs

Reply:
Put imply for some of the items. Note that MBEDTLS_USE_PSA_CRYPTO has no effect on TF-M image